system_tests/tests/pearlpc.py

```diff
@@ -261,9 +253,9 @@ def test_WHEN_limits_are_set_THEN_limits_update(self, _, pv):
     def test_WHEN_device_disconnected_THEN_limits_go_into_alarm(self, _, pv):
         self.ca.assert_that_pv_alarm_is(pv, self.ca.Alarms.NONE)
 
-        with self._disconnect_device():
+        with self.lewis.backdoor_simulate_disconnected_device():
             self.ca.assert_that_pv_alarm_is(pv, self.ca.Alarms.INVALID)
-
+        # Assert alarms clear on reconnection
         self.ca.assert_that_pv_alarm_is(pv, self.ca.Alarms.NONE)
```

Comment:
How do you feel about placing this into a separate private method or new test entirely?